### ISSUE TITLE
smarter npc ai

### ISSUE THREAD

**Issue by blablase** [CleverRaven/Cataclysm-DDA#77562]
### Is your feature request related to a problem? Please describe.

hate it when the npc burn through trillion of healing item just to get hit again.

make the npc smarter like hit and run rather than just face tanking the 3-5 zombies 

### Solution you would like.

something like a safe mode and an ability to see the future move of an enemy that prevent the npc to use healing item, if the future move of an enemy is in the range of the npc safe mode in the next 10min+ in game (player setting) npc won't use the healing item also make them wanted to use range weapon hit and run rather to waste healing item.

position them self in a way that they kill one at a time or just do hit and run, or fight in a place that have alot of high movement cost terrain and they know how to traverse it

**Comment by Holli-Git** [comment]
Making the NPC AI "smarter" is not a helpful feature request. Please be more descriptive in your describing your problem and solutions, it's very hard to parse what you're saying, and if it's a bug or feature. For what this feature request is about, having the ability to "see future moves of an enemy" is outside feasibility or reason, and as such this is closed.
